### PR TITLE
Remove disabled

### LINKED ISSUE #124: Enlever les options indisponibles
Les intégrateurs (qui se font de plus en plus nombreux !) n'ont aucun intérêt à afficher des options désactivées, qui peuvent être une source de confusion pour l'utilisateur (est-ce que j'ai mal remplis un des premiers champs pour ne pas avoir l'option salaire net, l'option salaire annuel ?). 

L'intérêt de ces champs est dans la démonstration des possibilités futures du module, il faudrait donc que l'on garde ces champs sur une page de démonstration (les cacher en CSS ?). 

Au sujet de l'option `par jour` déjà enlevée dans #121, il nous semble, dans les discussions du dernier open lab, que c'est la seule temporalité qui n'est pas mentionnée. L'heure et le mois sont régulièrement cités.

### PR DESCRIPTION
**Pull request by laem**
Resolves #124

### CODE REVIEW

File: assets/embauche.html [hunk 1]
```diff
@@ -34,18 +30,10 @@
...
 					</label>
 					<span class="input-help">Rémunération totale, dont primes. <br>Temps plein <em>(min. <span data-source="smic_proratise" data-round>1467</span>)</em> ou partiel. </span>
-
-					<select data-sets="#salaire@name">
-						<option value="salaire_de_base">brut</option>
-						<option disabled value="salaire_net_a_payer">net</option>
-					</select>
-
+					&nbsp;
+					<span>brut</span>
```
Comment (MattiSG) [inline assets/embauche.html]: Why wrap everything within `<span>`s?

File: assets/embauche.html [hunk 2]
```diff
@@ -321,6 +301,9 @@
 			</tr>
 		</table>
 
+		<button id="createTest"><a href="mailto:embauche@sgmap.fr?subject=Erreur dans les résultats du simulateur">Signaler une erreur</a></button>
+		<!--
 		<button id="createTest">Corriger ces résultats</button>
+		-->
```
Comment (MattiSG) [inline assets/embauche.html]: [Please, don't commit commented out code](https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f#.2777523y4).

Comment (laem) [inline assets/embauche.html]: Would you remove the whole `tests.js` file too then ?

File: assets/embauche.html (hunk 1 above)
Comment (laem) [inline assets/embauche.html]: The salary input hover won't be able to hide these texts if they're not inside spans.

File: assets/embauche.html (hunk 2 above)
Comment (MattiSG) [inline assets/embauche.html]: Yes.

File: assets/embauche.html (hunk 1 above)
Comment (MattiSG) [inline assets/embauche.html]: Ok, but do we need each _word_ to be in a separate `<span>`?  :wink:

File: css/details.css [hunk 3]
```diff
@@ -58,8 +58,13 @@ tr.imprecise td::after {
 	text-align: center;
 }
 
-details #createTest {
+#reportDetailsProblem {
```
Comment (MattiSG) [inline css/details.css]: Why `reportDetailsProblem` rather than `reportProblem`?

File: assets/embauche.html [hunk 4]
```diff
@@ -321,6 +299,6 @@
 			</tr>
 		</table>
 
-		<button id="createTest">Corriger ces résultats</button>
+		<button id="reportDetailsProblem"><a href="mailto:embauche@sgmap.fr?subject=Erreur dans les résultats du simulateur">Signaler une erreur</a></button>
```
Comment (MattiSG) [inline assets/embauche.html]: `<button>` should be [wrapped by](http://stackoverflow.com/questions/16280684) `<a>`. Or, even better, make it simply an `<a>` and style it with CSS to give it the look of a button if you want to have it look like a button.

File: css/details.css (hunk 3 above)
Comment (laem) [inline css/details.css]: This button concerns the detail section only

File: assets/embauche.html (hunk 4 above)
Comment (laem) [inline assets/embauche.html]: Nice one :fearful:

File: css/details.css (hunk 3 above)
Comment (MattiSG) [inline css/details.css]: Oh ok, got it  ;)

**Comment by laem** [comment]
Oops I fixed up a fix up !

**Comment by MattiSG** [comment]
GTM!

**Pull request merged**